dns-records.tf

```diff
@@ -402,6 +402,19 @@ resource "azurerm_dns_txt_record" "apex_jenkinsciorg" {
   tags = local.default_tags
 }
 
+resource "azurerm_dns_txt_record" "jetbrains_sponsorship_domain_verification" {
+  name                = "@"
```

Comment:
you need to update the existing one:

https://github.com/NotMyFault/azure-net/blob/db2c46611c1d985afe37c85825994501555de5c8/dns-records.tf#L360-L382

Reply:
Moved it upwards